Source/WebKit/GPUProcess/ios/GPUProcessIOS.mm

```diff
@@ -38,6 +38,10 @@
 namespace WebKit {
 using namespace WebCore;
 
+void GPUProcess::platformInitializeGPUProcess(GPUProcessCreationParameters&)
+{
+}
```

Comment:
Perhaps this should be in `Source/WebKit/GPUProcess/cocoa/GPUProcessCocoa.cpp` instead (creating that file)? So it can be shared with macOS and iOS?